ansible_runner/config/_base.py

```diff
@@ -262,7 +263,15 @@ def _prepare_env(self, runner_mode='pexpect'):
                 self.fact_cache = os.path.join(self.artifact_dir, self.settings['fact_cache'])
 
         # Use local callback directory
-        if not self.containerized:
+        if self.containerized:
+            # when containerized, copy the callback dir to $pirvate_data_dir/artifacts/<job_id>/callback
+            # than append to env['Ansible_CALLBACK_PLUGINS'] with the copyied location
+            callback_dir = os.path.join(self.artifact_dir, 'callback')
+            shutil.copytree(get_callback_dir(), callback_dir)
```

Comment:
I tested this and saw:

```
drwxrwxrwx. 3 alancoding alancoding 4096 May  2 15:53 callback
```

These permissions are going to set off alarm bells. We probably need to pre-create the directory so we can set specific permissions. Practically, we'd probably copy from above in this same file

https://github.com/ansible/ansible-runner/blob/8174803dc5cf2d2ab70fc90981b24a39ca7601ff/ansible_runner/config/_base.py#L138

but `os.mkdir` might be preferable here.

Reply:
discussed with @AlanCoding about the topic this inherited the file permission of the parent ansible-runner installation... so long as the permission for the original install is good the copied version would be good as well